diff --git a/Android.bp b/Android.bp
index a20fffb..36ba6ab 100644
--- a/Android.bp
+++ b/Android.bp
@@ -57,7 +57,7 @@ rust_test {
         "parser",
     ],
     rustlibs: [
-        "libbitflags",
+        "libbitflags-1.3.2",
         "libcriterion",
         "liblibc",
     ],
@@ -82,7 +82,7 @@ rust_test {
     ],
     rustlibs: [
         "libanes",
-        "libbitflags",
+        "libbitflags-1.3.2",
         "libcriterion",
         "liblibc",
     ],
@@ -101,7 +101,7 @@ rust_library {
         "parser",
     ],
     rustlibs: [
-        "libbitflags",
+        "libbitflags-1.3.2",
     ],
     apex_available: [
         "//apex_available:platform",
